README.md

```diff
@@ -63,7 +65,7 @@ How would I go about accessing the string "pizza" in the above array?
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+```scope refers to variables that any given function is able to "see" or access.
```

Comment:
Close- scope is not always addressed through the lens of functions.  Some helpful rules that address which variables are available where are: 

1. Variables defined without the var keyword are always placed in the global scope.
2. Variables that are defined using the var keyword are created in the current local scope.
3. Every function and only functions create a new local scope.
4. The current scope includes all outer (enclosing) scopes.